package com.google.android.gms.internal.p002firebaseauthapi;

import java.util.Comparator;

/* compiled from: r8-map-id-d9727bc7e00235ac513a61fce9aaf2b7a3ad41c0941d80eeaba89f3cd6d3eec5 */
/* loaded from: classes2.dex */
final class zzahq implements Comparator<zzaho> {
    @Override // java.util.Comparator
    public final /* synthetic */ int compare(zzaho zzahoVar, zzaho zzahoVar2) {
        zzaho zzahoVar3 = zzahoVar;
        zzaho zzahoVar4 = zzahoVar2;
        zzahu zzahuVar = (zzahu) zzahoVar3.iterator();
        zzahu zzahuVar2 = (zzahu) zzahoVar4.iterator();
        while (zzahuVar.hasNext() && zzahuVar2.hasNext()) {
            int iCompareTo = Integer.valueOf(zzaho.zza(zzahuVar.zza())).compareTo(Integer.valueOf(zzaho.zza(zzahuVar2.zza())));
            if (iCompareTo != 0) {
                return iCompareTo;
            }
        }
        return Integer.valueOf(zzahoVar3.zzb()).compareTo(Integer.valueOf(zzahoVar4.zzb()));
    }
}
